extract/src/main/resources/static/lang/de/rulesHelp.html

```diff
@@ -0,0 +1,481 @@
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>client</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Kunde, der das Anfrageelement bestellt hat</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>clientguid</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Identifikator des Kunden, der das Anfrageelement bestellt hat</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>tiers</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Dritter, für den das Anfrageelement bestellt wurde</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>tiersguid</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Identifikator des Dritten, für den das Anfrageelement bestellt wurde</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>surface</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Fläche des Umfangs des Anfrageelements in m²</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Numerisch</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>organism</p>
+            </td>
+
+            <td class="popover-table-data">
+                <p>Organismus des Kunden, der das Anfrageelement bestellt hat</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Attributiv</p>
+            </td>
+            <td class="popover-table-data">
+                <p>Text</p>
+            </td>
+        </tr>
+        <tr>
+            <td class="popover-table-data">
+                <p>organismguid</p>
```

Comment:
Is this actually a text that is displayed to the operator of Extract,
or is it a technical id which will be matched with something else? If so, with what?